server/server.js

```diff
@@ -69,6 +70,30 @@ app.get(
   }
 );
 
+app.get(
+  '/high-score',
+  sessionController.isLoggedIn,
+  scoreController.getHighScore,
+  (req, res) => {
+    if (!res.locals.cookieSessionMatch)
+      return res.status(200).json('Invalid session');
+    return res.status(200).json(res.locals);
```

Comment:
Wondering if the two `returns` should return an object of the same structure. The successful session match returns an object with a key of highScore, but the unsuccessful (which is actually still successful) is just a string. If you made them both the same structure, then could be easier to reason about on the frontend.

What do you think?

Reply:
An unsuccessful cookieSessionMatch means there is no valid session for the user. Ideally, this should redirect the user to the sign in page. However, didn't want to add this yet until front-end was able to send GET request to '/high-score' to test how it plays out.

Reply:
Sounds good.